src/components/PupilComponents/SignpostTeachersInlineBanner/SignpostTeachersInlineBanner.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  OakBox,
+  OakInlineBanner,
+  OakSecondaryLink,
+} from "@oaknational/oak-components";
+import { useRef, useState } from "react";
+import { Transition, TransitionStatus } from "react-transition-group";
+import styled from "styled-components";
+
+import useReferrer, { referrerSources } from "@/hooks/useReferrer";
+
+const durationInSeconds = 0.5;
```

Comment:
This was so the duration could be set in one place. I couldn't find any tokens or better method of setting this up. Also this component is only supposed to be used for a few weeks.